microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -763,17 +763,32 @@ def verify_serial_console_is_enabled(self, node: Node) -> None:
             isinstance(node.os, Ubuntu) and node.os.information.version >= "22.10.0"
         ):
             log_output = node.tools[Dmesg].get_output()
+            log_file = "dmesg"
         else:
+            cat = node.tools[Cat]
             if node.shell.exists(node.get_pure_path("/var/log/messages")):
                 log_file = "/var/log/messages"
+                log_output = cat.read(log_file, force_run=True, sudo=True)
             elif node.shell.exists(node.get_pure_path("/var/log/syslog")):
                 log_file = "/var/log/syslog"
+                log_output = cat.read(log_file, force_run=True, sudo=True)
             else:
+                log_file = "journalctl"
+                journalctl = node.tools[Journalctl]
+                journalctl_out = journalctl.run(
+                    "-b | head -n 1000",
```

Comment:
how we can make sure the str can be found in first 1000 lines

Reply:
I just tried for few distro's and it passed. Not sure if there is any way to guarantee. 
Without the head I thought the output would be too large. Do you recommend removing head -n 1000?

Reply:
if the vm was reused by many cases, probably the serial console config output is not within 1000 lines
if it is too large, can we try to grep it from the file? if it doesn't have a file. we can redirect the output into the file.

Reply:
@LiliDeng , but with -b option in journalctl, we will get logs starting from previous boot.
So, running multiple cases should not matter, right?

Reply:
yes, journalctl -b means the recent reboot.
we can move this into journalctl tool and make the line number as a parameter, then get the output from one method in journalctl